### PR TITLE
Add Safari real values for Media APIs

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR adds real values for Safari for various Media APIs based upon results from the mdn-bcd-collector project.

### CODE REVIEW

File: api/MediaStreamTrack.json [hunk 1]
```diff
@@ -849,10 +849,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```
Comment (sideshowbarker) [inline api/MediaStreamTrack.json:852]: Ghost feature?

File: api/MediaStreamTrack.json [hunk 2]
```diff
@@ -1041,10 +1041,10 @@
...
```
Comment (sideshowbarker) [inline api/MediaStreamTrack.json:1044]: Ghost feature?

**Review by sideshowbarker — COMMENTED** [review]
ghost features…

File: api/MediaStreamTrack.json (hunk 1 above)
Comment (vinyldarkscratch) [reply api/MediaStreamTrack.json:852]: There does seem to be a lot of these ghost features that we're finding...  I know that @saschanaz has been doing a little project to narrow down and remove these features without support!

Comment (foolip) [reply api/MediaStreamTrack.json:852]: This one's in https://w3c.github.io/webrtc-identity/identity.html#isolated-track, but I guess not implemented.

File: api/MediaStreamTrack.json (hunk 2 above)
Comment (foolip) [reply api/MediaStreamTrack.json:1044]: I think this was in Chrome at some point, but that's not reflected in the data.

**Review by foolip — APPROVED** [review]

**Pull request merged**